master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -2,41 +2,64 @@
 title: Kubeadm Hosted Install
```

Comment:
I am seeing kubeadm lowercased even when it begins a sentence in the k8s docs so we should probably do the same, as well. https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/